src/utils/fetch-config.ts

```diff
@@ -12,34 +12,13 @@ export async function fetchConfig (): Promise<void> {
   //
   const processEnvVueAppPath: string = process.env.VUE_APP_PATH // eg, cooperatives
   const processEnvBaseUrl: string = process.env.BASE_URL // eg, /cooperatives/
-  const windowLocationPathname: string = window.location.pathname // eg, /cooperatives/CP0000841/...
-  const windowLocationOrigin: string = window.location.origin // eg, http://localhost:8080
```

Comment:
Typing here is optional because the assigned value is already of type "string". Some Typescript linters will actually report a "unnecessary type annotation" error.